tests/unit/transformers/test_utils.py

```diff
@@ -225,6 +226,18 @@ def test_learn_rounding_digits_less_than_15_decimals():
     assert output == 3
 
 
+def test_learn_rounding_digits_pyarrow():
```

Comment:
can we also add one more using `floats` to make sure it actually learns the digits ?